iaso/api/org_unit_change_requests/serializers.py

```diff
@@ -157,3 +162,59 @@ class Meta:
 
     def get_new_groups(self, obj: OrgUnitChangeRequest):
         return [group.name for group in obj.new_groups.all()]
+
+
+class OrgUnitChangeRequestWriteSerializer(serializers.ModelSerializer):
+    """
+    Used to create or update one `OrgUnitChangeRequest` instance.
+    """
+
+    uuid = serializers.UUIDField(required=False, default=uuid.uuid4)
+    org_unit_id = serializers.PrimaryKeyRelatedField(
+        source="org_unit",
+        queryset=OrgUnit.objects.all(),
+        required=False,
+    )
+    new_parent_id = serializers.PrimaryKeyRelatedField(
```

Comment:
How do we validate the the new_parent is among the possible parent (same account, same version_id, ideally, no cycle in the parent links) ?

Reply:
This will be done in another PR ([IA-2494](https://bluesquare.atlassian.net/browse/IA-2494)).